CHANGELOG.md

```diff
@@ -381,6 +382,21 @@ Adding a new version? You'll need three changes:
 [KongIngress to KongUpstreamPolicy migration guide]: https://docs.konghq.com/kubernetes-ingress-controller/latest/guides/migrate/kongingress/
 [Migrate Credential Type Labels]: https://docs.konghq.com/kubernetes-ingress-controller/latest/guides/migrate/credential-kongcredtype-label/
 
+## [2.12.3]
+
+> Release date: 2023-12-19
+
+### Fixed
+
+- Fix(manager): set InitCacheSyncDuration to 5s by default and allow it to be configured via cli.
```

Comment:
Can we add the CLI flag name here as it should be used on the command line?

Reply:
```suggestion
- Fix(manager): set InitCacheSyncDuration to 5s by default and allow it to be configured via `-init-cache-sync-duration` CLI flag.
```

Is this ok?

Reply:
```suggestion
- Fix(manager): set InitCacheSyncDuration to 5s by default and allow it to be configured via `--init-cache-sync-duration` CLI flag
```

Using double dash, just like it's shown when calling KIC's binary with `-h` or `--help`